### DISCUSSION TITLE
Importing vs cloning a github repository

### DISCUSSION THREAD

**Discussion by illtellyoulater** [community/community#111423] [Repositories] [answered]
Could anyone please explain me if importing someone else github repository with the import function (rather than cloning it) will still allow me to pull new commits from the original repository?

**Comment by ngljcb** [comment, marked as answer]
When you import a repository from GitHub using the import function, it creates a separate copy of that repository in your account. This copy is not directly linked to the original repository, meaning you won't be able to pull new commits from the original repository using standard Git commands.

However, you can manually synchronize your repository with the original one by adding the original repository as a remote, fetching the latest changes from it, merging those changes into your local repository, and then pushing the updated code to your repository. This process allows you to keep your repository up-to-date with the latest changes from the original one. Just remember that if there are any conflicts during the merging process, you'll need to resolve them manually.
